docs/source/basics/customizing_interface.md

```diff
@@ -231,8 +48,16 @@ Each key in `metadata.parameters` must be the name of one of the parameters of t
 - `batch_table_column` (boolean): Whether this parameter should have a column to itself in the batch table at the top of the parameters page,
 - `appearance_type`: Either `LatchAppearanceType.line` or `LatchAppearanceType.paragraph`, which style to render text inputs as.
 - `rules`: A list of `LatchRule`s which consist of a regular expression and a message. If provided, an input must match all given regexes in order to appear valid in the front end - if it fails to match one of the regexes, the corresponding message is displayed.
+- `documentation`: A URL that leads to documentation for the workflow itself,
+- `author`: This must be a `LatchAuthor` objects, whose fields are:
```

Comment:
link to api docs instead

Reply:
use sphinx references where possible (e.g. `` {class}`~whatever.module.name.LatchAuthor` ``)
this entire section needs that

note you can also specify the text that gets used here, the default is the name of the object itself (and the `~` inside the backticks means unqualified name)